Assets/MixedRealityToolkit.Services/InputSystem/GazeProvider.cs

```diff
@@ -517,12 +518,21 @@ public void UpdateEyeGaze(IMixedRealityEyeGazeDataProvider provider, Ray eyeRay,
             Timestamp = timestamp;
         }
 
+        public void UpdateEyeTrackingStatus(IMixedRealityEyeGazeDataProvider provider, bool userIsEyeCalibrated)
+        {
+            this.isEyeCalibrated = userIsEyeCalibrated;
+        }
+
         /// <summary>
         /// Ensure that we work with recent Eye Tracking data. Return false if we haven't received any
         /// new Eye Tracking data for more than 'maxETTimeoutInSeconds' seconds.
         /// </summary>
         private bool IsEyeTrackingAvailable => (DateTime.UtcNow - latestEyeTrackingUpdate).TotalSeconds <= maxEyeTrackingTimeoutInSeconds;
 
+        /// <summary>
+        /// Boolean to check whether the user went through the eye tracking calibration.
+        /// </summary>
+        public bool? IsEyeCalibrationValid => isEyeCalibrated;
```

Comment:
Do we expect that someone would treat the "null" state any different from the false state? i.e. effectively I'm trying to understand that this API is now a tristate - i.e. null (which currently isn't defined), false (means not calibrated) and true (calibrated). The question someone might have is, what does null mean? Can things go back to null? Should a consumer really just care about true false?

Reply:
The problem here was that at the startup of the app, the default would be to return false, but that would trigger the event of "not being calibrated" to fire. Null allows for us to know whether the variables has been initialized appropriately and has received an actual value from the eye tracking system.

Reply:
I guess I could have also set the default to "isEyeCalibrated = true", but seems a little less clean to me.

Reply:
No worries - I think it would be good just to have it commented saying that "hey, a null value means such and such" otherwise it's not clear as a reader why it's not really a bool, but actually a tri-state (and thus, by the typical tristate boolean can't go back to UNDEFINED/null, but again it's just a matter of creating clarity for readers without making them have to dig through the implementation to understand the behavior)

Reply:
Sounds reasonable. I'll add a comment.